src/app/components/Curation/index.tsx

```diff
@@ -81,6 +89,8 @@ const Curation = ({
           headingBackgroundColour={GHOST}
         />
       );
+    case RADIO_SCHEDULE:
+      return <RadioSchedule initialData={radioSchedule} />;
```

Comment:
I am just thinking ahead to how we might test this. I'd like for us to get it onto the test environment ASAP, so that we can all have a play around with it. 

Could we add a `!isLive()` check here, so that even if we merge the PR, it won't go straight to live? We would like to give editorial the chance to try it out on one of the homepages first, to see what it looks like, and if all is well, then we would remove the check in a separate PR. 
```suggestion
      return !isLive() && <RadioSchedule initialData={radioSchedule} />;
```